### PR TITLE
Add test cases for methods

### PR DESCRIPTION
**Pull request by XihuaZ**

### CODE REVIEW

File: src/main/java/seedu/address/ui/MainWindow.java [hunk 1]
```diff
@@ -177,10 +180,6 @@ private void handleExit() {
         primaryStage.hide();
     }
 
-    public EmployeeListPanel getEmployeeListPanel() {
```
Comment (SungMatt) [inline src/main/java/seedu/address/ui/MainWindow.java:180]: I'm not too familiar with the UI -- is it safe to remove @starrylight99 ?

**Review by SungMatt — COMMENTED** [review]
Rest looks good! Just pending the comment on the UI!

File: src/main/java/seedu/address/ui/MainWindow.java (hunk 1 above)
Comment (XihuaZ) [reply src/main/java/seedu/address/ui/MainWindow.java:180]: It is a deprecated method, and was used in a very [old version](https://github.com/AY2324S1-CS2103-T16-1/tp/commit/b8ed0900daea2e5e2f771cd4453099a9eaa6ce19#diff-4237a1e00a40f25992ac3d6b9a02b9a32bf6d8ab5dac0ac7bddd1de25a896d2d) implementation of the UI. It is safe to delete.

Comment (SungMatt) [reply src/main/java/seedu/address/ui/MainWindow.java:180]: Ok ok!

**Review by SungMatt — APPROVED** [review]
Thanks for all the hard work Xihua! LGTM!

**Pull request merged**